app/instance-initializers/bugsnag.js

```diff
@@ -42,6 +44,15 @@ export default {
 	},
 
 	_onError(error) {
+
+		if (!error || this.errorFilter.includes(error.name)) {
```

Comment:
This is not good, you should call the function directly.

```suggestion
		if (!error || !appMethods.errorFilter(error)) {
```